packages-exp/app-check-exp/src/client.ts

```diff
@@ -0,0 +1,134 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  BASE_ENDPOINT,
+  EXCHANGE_DEBUG_TOKEN_METHOD,
+  EXCHANGE_RECAPTCHA_TOKEN_METHOD
+} from './constants';
+import { FirebaseApp } from '@firebase/app-exp';
+import { ERROR_FACTORY, AppCheckError } from './errors';
+import { Provider } from '@firebase/component';
+import { AppCheckTokenInternal } from './types';
+
+/**
+ * Response JSON returned from AppCheck server endpoint.
```

Comment:
I think we would normally say "JSON response," correct?

Reply:
I think this makes sense for us and it's not public.

Reply:
SG, thanks!